support/static/scripts/payments.js

```diff
@@ -459,7 +414,7 @@
     start = start ? start : Date.now();
     const endStates = ['succeeded', 'processing', 'canceled'];
     // Retrieve the PaymentIntent status from our server.
-    const rawResponse = await fetch(`payment_intents/${paymentIntent}/status`);
+    const rawResponse = await fetch(`support/confirm_payment/${paymentIntent}/status`);
```

Comment:
URL here should be absolute, not relative. This would make this future-proof and also consistent with other `fetch` invocations